README.md

```diff
@@ -535,7 +536,6 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 - [Paper Clip](https://apps.gnome.org/PdfMetadataEditor) - PDF metadata editor `#vala` `#gtk4` `#libadwaita` `#gnome`.
 - [PDF Arranger](https://github.com/pdfarranger/pdfarranger) - PDF editor with merging, splitting, rotating, cropping and rearranging based on [pikepdf](https://github.com/pikepdf/pikepdf) `#python` `#gtk3` `#libhandy`.
-- [PDF Slicer](https://junrrein.github.io/pdfslicer) - Simple application to extract, merge, rotate and reorder pages of PDF documents with undo/redo support `#c++` `#gtk3`.
```

Comment:
Why should this application be removed?

Reply:
4 years without any updates

Reply:
I guess we can remove it since PDF Arranger provides the same functionnality and is up to date.